Plugins/Flow.Launcher.Plugin.Explorer/Settings.cs

```diff
@@ -32,6 +32,7 @@ public class Settings
 
         public bool ShowWindowsContextMenu { get; set; } = true;
 
+        public bool DefaultOpenInFileManager { get; set; } = false;
```

Comment:
should this be called "DefaultOpenFolderInFileManager" or just "OpenFolderInFileManager"